mLRS/Common/fhss.h

```diff
@@ -56,8 +56,53 @@ const uint32_t fhss_freq_list_433[] = {
     SX12XX_FREQ_MHZ_TO_REG(433.480),
 };
 
-const uint8_t fhss_bind_channel_list_433[] = {
-    0, // just pick some
+#ifdef FHSS_HAS_CONFIG__70_CM_HAM
+// USA Ham Bands w/ Tech License
+// https://www.arrl.org/files/file/Tech%20Band%20Chart/US%20Amateur%20Radio%20Technician%20Privileges.pdf
+// 33 Channels w/ 0.6 MHz spacing
+
+const uint32_t fhss_freq_list_70_CM_HAM[] = {
```

Comment:
here it should be all lower case, _70_cm_ham

Reply:
Done.